### PR TITLE
Assignment 1 added

### PR DESCRIPTION
**Pull request by sharyu04**
1. Write a program to calculate the simple interest 
First-line has the comma-separated values of the Principal, rate and time (in years) respective
*constraints: Round simple interest float value to 2 decimal places
2. Write a program to calculate the area of the circle, First line has a value of the radius of the circle
constraint
1. Use const PI from the package math package
2. Use the Pow function from the math package
3. Round area float value to 2 decimal places

### CODE REVIEW

File: Assgn1/Q1.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+package main
+
+import "fmt"
+
+func main() {
+	principal := 1000
```
Comment (Deepak1124) [inline Assgn1/Q1.go]: Please take input from user

File: Assgn1/Q1.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	ROI := 5
+	Time := 3
```
Comment (Deepak1124) [inline Assgn1/Q1.go]: Use proper functions(from main function) to call business logic

File: Assgn1/Q1.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	SI := (principal * ROI * Time)/100
+	ans := fmt.Sprintf("%.2f",float64(SI))
+	fmt.Println("Simple Interest = ", ans)
```
Comment (Deepak1124) [inline Assgn1/Q1.go]: Use fmt.Print() instead, to do proper formatting

File: Assgn1/Q2.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var radius float64
```
Comment (Deepak1124) [inline Assgn1/Q2.go:9]: Please take input from user

File: Assgn1/Q2.go [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+	var area float64
+	radius = 2
+	area = math.Pi * math.Pow(radius, 2)
```
Comment (Deepak1124) [inline Assgn1/Q2.go:17]: Use proper functions(from main function) to call business logic

File: Assgn1/Q2.go [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+	ans := fmt.Sprintf("%.2f",area)
+	fmt.Println("Area = ",ans)
```
Comment (Deepak1124) [inline Assgn1/Q2.go]: Use fmt.Print() instead, to do proper formatting

**Review by Deepak1124 — CHANGES_REQUESTED** [review]

File: Assgn1/Q1.go (hunk 1 above)
Comment (sharyu04) [reply Assgn1/Q1.go]: The changes are done

File: Assgn1/Q1.go (hunk 2 above)
Comment (sharyu04) [reply Assgn1/Q1.go]: The changes are done

File: Assgn1/Q1.go (hunk 3 above)
Comment (sharyu04) [reply Assgn1/Q1.go]: Changes done

File: Assgn1/Q2.go (hunk 4 above)
Comment (sharyu04) [reply Assgn1/Q2.go:9]: Changes are done

File: Assgn1/Q2.go (hunk 5 above)
Comment (sharyu04) [reply Assgn1/Q2.go:17]: Changes done

File: Assgn1/Q2.go (hunk 6 above)
Comment (sharyu04) [reply Assgn1/Q2.go]: Changes done

**Pull request merged**